WKData/Sources/WKData/Extensions/URL+API.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+extension URL {
+    static func mediaWikiAPIURL(project: WKProject) -> URL? {
+        var components = URLComponents()
+        components.scheme = "https"
+        components.path = "/w/api.php"
```

Comment:
Would it make sense to generalize out these repeated constructions into a `WMFAPIPath`-ish from `NSURL+WMFLinkParsing` thing (`BaseMediaWikiAPIPathComponents`?)?

Reply:
Done (I think) - let me know if it's what you had in mind!

Reply:
To clarify, I think even pulling the repeated scheme would be useful. Like:

```swift
private static var baseMediaWikiAPIPathComponents: URLComponents {
	var components = URLComponents()
	components.scheme = "https"
	components.path = "/w/api.php"
	return components
}

static func mediaWikiAPIURL(project: WKProject) -> URL? {
    var components = baseMediaWikiAPIPathComponents

    switch project {
    ...
}
```

Reply:
Sorry I missed this one - will make a note to put it in a later PR. Thanks!